src/main/java/org/usfirst/frc4904/robot/commands/controlpanel/ColorSensor.java

```diff
@@ -0,0 +1,223 @@
+          /`-.                                  `.  /   /
+         :    `-  ...........,                   | /  .'
+         |         ``:::::::'       .            |<    `.
+         |             ```          |           x| \ `.:``.
+         |                         .'    /'   xXX|  `:`M`M':.
+         |    |                    ;    /:' xXXX'|  -'MMMMM:'
+         `.  .'                   :    /:'       |-'MMMM.-'
+          |  |                   .'   /'        .'MMM.-'
+          `'`'                   :  ,'          |MMM<
+            |                     `'            |tbap\
+             \                                  :MM.-'
+              \                 |              .''
+               \.               `.            /
+                /     .:::::::.. :           /
+               |     .:::::::::::`.         /
+               |   .:::------------\       /
+              /   .''               >::'  /
+              `',:                 :    .'
+
+``````¶0````1¶1_```````````````````````````````````````
+```````¶¶¶0_`_¶¶¶0011100¶¶¶¶¶¶¶001_````````````````````
+````````¶¶¶¶¶00¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶0_````````````````
+`````1_``¶¶00¶0000000000000000000000¶¶¶¶0_`````````````
+`````_¶¶_`0¶000000000000000000000000000¶¶¶¶¶1``````````
+```````¶¶¶00¶00000000000000000000000000000¶¶¶_`````````
+````````_¶¶00000000000000000000¶¶00000000000¶¶`````````
+`````_0011¶¶¶¶¶000000000000¶¶00¶¶0¶¶00000000¶¶_````````
+```````_¶¶¶¶¶¶¶00000000000¶¶¶¶0¶¶¶¶¶00000000¶¶1````````
+``````````1¶¶¶¶¶000000¶¶0¶¶¶¶¶¶¶¶¶¶¶¶0000000¶¶¶````````
+```````````¶¶¶0¶000¶00¶0¶¶`_____`__1¶0¶¶00¶00¶¶````````
+```````````¶¶¶¶¶00¶00¶10¶0``_1111_`_¶¶0000¶0¶¶¶````````
+``````````1¶¶¶¶¶00¶0¶¶_¶¶1`_¶_1_0_`1¶¶_0¶0¶¶0¶¶````````
+````````1¶¶¶¶¶¶¶0¶¶0¶0_0¶``100111``_¶1_0¶0¶¶_1¶````````
+```````1¶¶¶¶00¶¶¶¶¶¶¶010¶``1111111_0¶11¶¶¶¶¶_10````````
+```````0¶¶¶¶__10¶¶¶¶¶100¶¶¶0111110¶¶¶1__¶¶¶¶`__````````
+```````¶¶¶¶0`__0¶¶0¶¶_¶¶¶_11````_0¶¶0`_1¶¶¶¶```````````
+```````¶¶¶00`__0¶¶_00`_0_``````````1_``¶0¶¶_```````````
+``````1¶1``¶¶``1¶¶_11``````````````````¶`¶¶````````````
+``````1_``¶0_¶1`0¶_`_``````````_``````1_`¶1````````````
+``````````_`1¶00¶¶_````_````__`1`````__`_¶`````````````
+````````````¶1`0¶¶_`````````_11_`````_``_``````````````
+`````````¶¶¶¶000¶¶_1```````_____```_1``````````````````
+`````````¶¶¶¶¶¶¶¶¶¶¶¶0_``````_````_1111__``````````````
+`````````¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶01_`````_11____1111_```````````
+`````````¶¶0¶0¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶1101_______11¶_```````````
+``````_¶¶¶0000000¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶0¶0¶¶¶1````````````
+`````0¶¶0000000¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶1`````````````
+````0¶0000000¶¶0_````_011_10¶110¶01_1¶¶¶0````_100¶001_`
+```1¶0000000¶0_``__`````````_`````````0¶_``_00¶¶010¶001
+```¶¶00000¶¶1``_01``_11____``1_``_`````¶¶0100¶1```_00¶1
+``1¶¶00000¶_``_¶_`_101_``_`__````__````_0000001100¶¶¶0`
+``¶¶¶0000¶1_`_¶``__0_``````_1````_1_````1¶¶¶0¶¶¶¶¶¶0```
+`_¶¶¶¶00¶0___01_10¶_``__````1`````11___`1¶¶¶01_````````
+`1¶¶¶¶¶0¶0`__01¶¶¶0````1_```11``___1_1__11¶000`````````
```

Comment:
@FlyN-Nick I swear to god if you paste another pikachu into our code you're off the team :)